modules/standard/BigInteger.chpl

```diff
@@ -516,56 +516,32 @@ module BigInteger {
       return (dbl: real, exp.safeCast(uint(32)));
     }
 
-    proc get_str(base: int = 10) : string {
+    // private method
+    @chpldoc.nodoc
+    proc getStr(base: int = 10): string {
```

Comment:
Should we make this a `private proc`?

Reply:
That unfortunately requires it no longer being a method